module/VuFind/src/VuFind/View/Helper/Root/ResultFeed.php

```diff
@@ -250,11 +260,88 @@ protected function getDcDate($record)
         return '';
     }
 
+    /**
+     * Get a link to the record using the record linker. Returns null if no
+     * link is available.
+     *
+     * @param RecordDriver $record Record to add to feed
+     *
+     * @return ?string
+     */
+    protected function getLinkFromRecordLinker(RecordDriver $record): ?string
+    {
+        $serverUrl = $this->getView()->plugin('serverurl');
+        $recordLinker = $this->getView()->plugin('recordLinker');
+        try {
+            $url = $serverUrl($recordLinker->getUrl($record));
+        } catch (\Laminas\Router\Exception\RuntimeException $e) {
+            // Not every record driver has a route defined; return null if the
+            // record linker fails to give us anything usable.
+            $url = null;
+        }
+        return $url;
+    }
+
+    /**
+     * See if we can find a usable link in the record driver itself. Returns
+     * null if no link is available.
+     *
+     * @param RecordDriver $record Record to add to feed
+     *
+     * @return ?string
+     */
+    protected function getLinkFromRecordDriverMethod(RecordDriver $record): ?string
```

Comment:
How about just `getLinkFromRecordDriver`? Since it's calling different methods, 'Method' isn't accurate, and it doesn't convey much information anyway.